### PR TITLE
third_party: vendor in go-wrapper

### PR DESCRIPTION
**Pull request by zombiezen**

### CODE REVIEW

**Comment by zombiezen** [comment]
Do not submit until the CI configs are updated.

**Comment by zombiezen** [comment]
@broady @shantuo Please review.

File: README.md [hunk 1]
```diff
@@ -3,5 +3,49 @@ golang-docker
...
+### Vendoring
+
+Your application will be built using Go 1.6 inside the docker instance without
+the experimental vendoring support.  To enable Go 1.6 experimental vendoring
+support, set the environment variable like this:
+
+        FROM gcr.io/google_appengine/golang
+        ENV GO15VENDOREXPERIMENT 1
+        COPY . /go/src/app
+        RUN go-wrapper install -tags appenginevm
```
Comment (broady) [inline README.md]: We can remove this now.

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.git
+**/.gitignore
+**/.DS_Store
+test.bash
```
Comment (broady) [inline .dockerignore]: No need, since everything is referred to specifically when using COPY

File: README.md (hunk 1 above)
Comment (zombiezen) [inline README.md]: This particular line, or the whole section?

File: .dockerignore (hunk 2 above)
Comment (zombiezen) [inline .dockerignore]: Not needed as we're keeping in base, but it does affect how much data is sent to the build server.

File: README.md (hunk 1 above)
Comment (broady) [inline README.md]: The whole section. Since we're on 1.6 now, GO15VENDOREXPERIMENT isn't worth a mention now.

**Comment by zombiezen** [comment]
Done. PTAL.

**Comment by zombiezen** [comment]
Ping @broady

**Pull request merged**